pontoon/settings/base.py

```diff
@@ -168,6 +172,7 @@ def _site_url():
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
     'django.middleware.security.SecurityMiddleware',
     'csp.middleware.CSPMiddleware',
+    'pontoon.base.middleware.AutomaticLoginUserMiddleware',
```

Comment:
nit: f you would override this particular setting on the demo instance the AUTO_LOGIN flag wouldn't be required.